dame_flame/grouped_mr.py

```diff
@@ -81,6 +80,5 @@ def algo2_GroupedMR(df_all, df_unmatched, covs_match_on, all_covs, treatment_col
             # store the bi in a column with df_all and also a column for "pair" with another
             # persons unit id. 
             # don't update that when someone gets added to an auxiliary matched group
-            # then at the end, iterate through it and create the nicely formatteed output. 
-        
-    return matched_rows, return_groups
\ No newline at end of file
+            # then at the end, iterate through it and create the nicely formatteed output.
+    return matched_rows, return_groups, all_units_in_g
```

Comment:
Need to update the header of this function to reflect the new parameter being returned from the function